docker/docker-compose.yml

```diff
@@ -10,10 +10,6 @@ services:
       - 6600:6600
       - 8800:8800
     restart: unless-stopped
-    volumes:
-      - ../shared/audiofolders:/home/pi/RPi-Jukebox-RFID/shared/audiofolders:rw
-      - ../shared/playlists:/root/.config/mpd/playlists:rw
-      - ./config/docker.mpd.conf:/root/.config/mpd/mpd.conf:rw
```

Comment:
Why do you move them out of the global `docker-compose` and duplicate them in the individual ones? Since they are the same in all OS docker compose, they can be unified here.

Reply:
They are different in the linux-as-host configuration, they are mapped to /home/pi instead to /root because the linux config uses the pi user in combination with the running users UID to connect to the pulseaudio socket.

Reply:
But they seem to be the same in Windows and Mac. You could use them as default in the `docker-compose.yml` and only overwrite them in `docker-compose-linux.yml`. This is how it works.

Reply:
Done; now `docker-compose.windows.yml` is completely empty so I removed it